corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
@benrudolph I think you'll want to use something like https://github.com/benrudolph/commcare-hq/blob/forms-bulk-144444/corehq/apps/data_interfaces/dispatcher.py#L51

If you're adding a feature for bulk form management, I'd suggest adding a Role ```BULK_CASE_MANAGEMENT```